dbt/include/databricks/macros/catalog.sql

```diff
@@ -18,3 +18,86 @@
     use catalog {{ adapter.quote(catalog) }}
   {% endcall %}
 {% endmacro %}
+
+{% macro databricks__get_catalog_relations(information_schema, relations) -%}
+
+    {% set query %}
+        with tables as (
+            {{ databricks__get_catalog_tables_sql(information_schema) }}
+            {{ databricks__get_catalog_relations_where_clause_sql(relations) }}
+        ),
+        columns as (
+            {{ databricks__get_catalog_columns_sql(information_schema) }}
+            {{ databricks__get_catalog_relations_where_clause_sql(relations) }}
+        )
+        {{ databricks__get_catalog_results_sql() }}
+    {%- endset -%}
+
+  {{ return(run_query(query)) }}
+    {{ return(run_query(query)) }}
+
```

Comment:
extra call to return?

Reply:
Good catch.  certainly a copy-paste issue that didn't cause tests to fail.